.env

```diff
@@ -0,0 +1,88 @@
+SELLER_ID=""
+SELLER_CLIENT_ID=""
+BUYER_CLIENT_ID=""
+TRANSLATE_BLOB_URL="http://127.0.0.1:10000/devstoreaccount1/ngx-translate/i18n/"
+BLOB_STORAGE_URL="http://127.0.0.1:10000/devstoreaccount1"
+
+# Middleware - these settings are based on ./src/Middleware/src/Headstart.Common/AppSettingConfigTemplate.json (colon character is not allowed in environment variable name)
+
+ApplicationInsightsSettings_InstrumentationKey=""
+
+AvalaraSettings_AccountID="0"
+AvalaraSettings_CompanyCode=""
+AvalaraSettings_CompanyID="0"
+AvalaraSettings_LicenseKey=""
+AvalaraSettings_BaseApiUrl="https//sandbox-rest.avatax.com/api/v2"
+
+BlobSettings_ConnectionString=""
+BlobSettings_HostUrl=""
+BlobSettings_Key=""
+BlobSettings_ContainerNameQueue="queue"
+BlobSettings_ContainerNameCache="cache"
+BlobSettings_ContainerNameExchangeRates="currency"
+BlobSettings_ContainerNameTranslations="ngx-translate"
+CMSSettings_BaseUrl="https//ordercloud-cms-test.azurewebsites.net"
+
+CardConnectSettings_Authorization=""
+CardConnectSettings_AuthorizationCad=""
+CardConnectSettings_BaseUrl="cardconnect.com"
+CardConnectSettings_CadMerchantID=""
+CardConnectSettings_Site="fts-uat"
+CardConnectSettings_UsdMerchantID=""
+
+CosmosSettings_DatabaseName=""
+CosmosSettings_EnableTcpConnectionEndpointRediscovery="false"
+CosmosSettings_EndpointUri="http://localhost"
+CosmosSettings_PrimaryKey="MQ=="
+CosmosSettings_RequestTimeoutInSeconds="15"
+
+EasyPostSettings_APIKey=""  # Shipping Calculation
+EasyPostSettings_FreeShippingTransitDays="3"
+EasyPostSettings_NoRatesFallbackCost="20"
+EasyPostSettings_NoRatesFallbackTransitDays="3"
+
+EnvironmentSettings_AppTimeoutInSeconds="30"
+EnvironmentSettings_BuildNumber=""
+EnvironmentSettings_Commit=""
+EnvironmentSettings_Environment="Test"
+
+FlurlSettings_TimeoutInSeconds="40"
+
+OrderCloudSettings_ApiUrl="https//sandboxapi.ordercloud.io"
+OrderCloudSettings_MiddlewareClientID=""
+OrderCloudSettings_MiddlewareClientSecret=""
+OrderCloudSettings_IncrementorPrefix="DB_TEST"
+OrderCloudSettings_WebhookHashKey=""
+
+SendGridSettings_ApiKey=""
+SendgridSettings_FromEmail=""
+SendgridSettings_CriticalSupportEmails=""
+SendgridSettings_SupportCaseEmail=""
+SendgridSettings_BillingEmail=""
+SendgridSettings_OrderSubmitTemplateID=""
+SendgridSettings_OrderApprovalTemplateID=""
+SendgridSettings_LineItemStatusChangeTemplateID=""
+SendgridSettings_QuoteOrderSubmitTemplateID=""
+SendgridSettings_NewUserTemplateID=""
+SendgridSettings_ProductInformationRequestTemplateID=""
+SendgridSettings_PasswordResetTemplateID=""
+SendgridSettings_CriticalSupportTemplateID=""
+
+SmartyStreetSettings_AuthID="" # Address validation
+SmartyStreetSettings_AuthToken=""
+SmartyStreetSettings_RefererHost=""
+SmartyStreetSettings_WebsiteKey=""
+
+ZohoSettings_AccessToken=""
+ZohoSettings_ClientId=""
+ZohoSettings_ClientSecret=""
+ZohoSettings_OrgID=""
+ZohoSettings_PerformOrderSubmitTasks="false"
```

Comment:
Isn't a .env file usually omitted from source control so that a user's local settings don't accidentally get pushed up?

Does it make sense to have a .env.template that is committed to source control and then have instructions for the user to create a .env file from that template?

Reply:
Good point, updated file and instructions.